homework/web_scrapping1.py

```diff
@@ -0,0 +1,38 @@
+import requests
+from bs4 import BeautifulSoup 
+import sqlite3
+
+conn= sqlite3.connect('cputest.db')
+c=conn.cursor()
+
+c.execute('''CREATE TABLE pythonblog( title TEXT, link TEXT)''')
```

Comment:
@sampleharvgithub Please don't include create table commands in the script.
Script should include only those things which wouldn't break the code.
Having create table/database commands in the script would force the script to create table/database per run which would throw errors if the table/database already exists .

Reply:
I had commented the create table command initially. But I tried to modify the program by including a command and it resulted in an error saying the table doesn't exist. Therefore I removed the comment. I forgot to comment the command while pushing the file to Git.

Reply:
@sampleharvgithub Please make the appropriate changes as required.